src/sdk/src/store_sdk.rs

```diff
@@ -44,14 +44,39 @@ impl StoreSDK {
         Self {
             client,
             signer,
-            session: SessionManager::new(),
+            session_pool: SessionPool::new(),
         }
     }
 
-    pub async fn restart_session(&mut self) -> std::result::Result<(String, i32), Status> {
-        match self.validate_query_session() {
-            Ok(_) => {
-                let query_session_info = self.session.get_session_info();
+    pub async fn open_session(
+        &mut self,
+        addr: &AccountAddress,
```

Comment:
I think addr is useless

Reply:
Yep. But How to gen a signature if we don't have any field in the request?